physx.js

```diff
@@ -891,17 +892,14 @@ const physxWorker = (() => {
     const uvsTypedArray = allocator.alloc(Float32Array, numUvs)
     uvsTypedArray.set(uvs)
 
-    const facesTypedArray = allocator.alloc(Uint32Array, numFaces)
-    facesTypedArray.set(faces)
-
-    const positionTypedArray = allocator.alloc(Float32Array, 3)
-    positionTypedArray.set(position)
+    let facesTypedArray;
+    if (isIndexed) {

```

Comment:
Btw this could be done with just passing faces === null.

Reply:
Modified https://github.com/webaverse/app/pull/2954
At js side, I check if `faces` is falsy, is it ok? Or need to input exact 'null' ?